### PR TITLE
Drafting/output

### PR DESCRIPTION
**Pull request by Leafrancia**
Draft of the output part of the project proposal for the YOS award

### CODE REVIEW

**Comment by robertomediavilla** [comment]
First section ("Benefits") edited. The second sub-section ("Lessons learnt") is not focused. You can keep the introduction in that subsection but then you need to move to list actual lessons learnt, e.g., how hard it is to write a manuscript using GitHub (and why it is important), importance of sharing preprints on ResearchGate for instance, etc.

File: output/Form.md [hunk 1]
```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
 -   [*Benefits*]{.underline}*: What benefits were realised (to date) through
```
Comment (DaniMori) [inline output/Form.md:65]: Please @Leafrancia , note there is only one text field (where it says `<!--# blank field -->`) so all the text should go there.

File: output/Form.md [hunk 2]
```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
...
     this initiative, and for whom?*
 
+As a leading international research group and a World Health Organization's
```
Comment (DaniMori) [inline output/Form.md]: I'm not sure if we can call ourselves "leading research group"...

File: output/Form.md [hunk 3]
```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
...
+collaborating centre, we work daily with researchers and professionals living
+abroad. This requires tools that help tracking exchanges between labs in a
+comprehensive way, such as statistical code, manuscript drafts, or meeting
+minutes. By taking this approach, we have not only foster a stronger open
+science practice within our lab, but also across different research consortia in
+the European Union (EU). This may potentially spread through different
```
Comment (DaniMori) [inline output/Form.md]: This sounds great and is very well written. However, I have the concern that this is not true, at least up to now, Remember we are talking here about "what we have achieved so far".

File: output/Form.md [hunk 4]
```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
 -   [*Benefits*]{.underline}*: What benefits were realised (to date) through
     this initiative, and for whom?*
 
+As a leading international research group and a World Health Organization's
+collaborating centre, we work daily with researchers and professionals living
+abroad. This requires tools that help tracking exchanges between labs in a
+comprehensive way, such as statistical code, manuscript drafts, or meeting
+minutes. By taking this approach, we have not only foster a stronger open
```
Comment (DaniMori) [inline output/Form.md]: "not only _fostered_..."

**Review by robertomediavilla — CHANGES_REQUESTED** [review]
First section ("Benefits") edited. The second sub-section ("Lessons learnt") is not focused. You can keep the introduction in that subsection but then you need to move to list actual lessons learnt, e.g., how hard it is to write a manuscript using GitHub (and why it is important), importance of sharing preprints on ResearchGate for instance, etc.

File: output/Form.md [hunk 5]
```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
...
+science practice within our lab, but also across different research consortia in
+the European Union (EU). This may potentially spread through different
+institutes in the region and improve reproducibility rates for the whole
+research community.For example,in the EU-funded project RESPOND, we have already
```
Comment (DaniMori) [inline output/Form.md]: "community. For example" (space after the point)

Comment (DaniMori) [inline output/Form.md]: Has that article used Github actually? If you want an example that uses it, you can link this repo: https://github.com/CCOMS-UAM/mapping-initiatives

Comment (DaniMori) [reply output/Form.md]: Also, mind these sentences dismissed from the "Description" section:

https://github.com/CCOMS-UAM/yos-award/blob/29e6f2eff0a4952bc8a02a103b6d237f2c23602b/output/Form.md?plain=1#L65

File: output/Form.md [hunk 6]
```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
...
+published open-access manuscripts to inform mental and occupational health
+policies for vulnerable populations during the COVID-19 pandemic, especially
+HCWs. Soon, we will report the RCT results and make the code available. This
+will allow researchers to have full access not only to the results of the study
+but also to how the study developed.
```
Comment (DaniMori) [inline output/Form.md]: Great! This is exactly what I try to convey in "Description"

File: output/Form.md [hunk 7]
```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
...
 -   [*Lessons learnt*]{.underline}*: What knowledge have you gained (so far)
     through your work?*
 
-<!--# blank field, max 400 words -->
+In the context of the COVID-19 pandemic, preprints increased, journals had
+open-access policies, etc. In a rapidly changing world, with the climate crisis,
+wars, etc., research should (and will) move rapidly and collaboratively. This
+requires training and collaboration, which is why we apply for this grant. One
+of the main benefits of developing this practice is having a complete history
+accessible to all. This increases the transparency of a study and facilitates
```
Comment (DaniMori) [inline output/Form.md]: Excellent! Mind also these lines dismissed from "Description":

https://github.com/CCOMS-UAM/yos-award/blob/29e6f2eff0a4952bc8a02a103b6d237f2c23602b/output/Form.md?plain=1#L58

File: output/Form.md [hunk 8]
```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
...
+communication. Moreover, it is an interesting tool for the researcher himself.
+Indeed, having this history allowed us to keep in writing what was done, what
+worked but also what led to an error. This written trace allowed us to save time
```
Comment (DaniMori) [inline output/Form.md]: "what led _to error_"

File: output/Form.md [hunk 9]
```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
...
+and avoid reproducing the same mistake while understanding why. Also, when we
+want to remember the previous steps and what has already been done by our peers,
+it is easy to navigate within it once familiar with the platform and find this
+information.
```
Comment (DaniMori) [inline output/Form.md]: This is great! I would also add something about the use of Opal (you may need help/input from @robertomediavilla and/or @Cristrinaranjus for that).

Also, something about the challenges of using these technologies would be in place. That would help us justify the "Follow-up" section (investing in training courses)

File: output/Form.md [hunk 10]
```diff
@@ -23,10 +24,39 @@ title: Yerun Open Science Award form
...
+
+<!--# blank field, 321 words -->
```
Comment (DaniMori) [inline output/Form.md]: Leave only word count, i.e.:

 ```
<!--# 321 words -->
```

**Review by DaniMori — CHANGES_REQUESTED** [review]
Well done @Leafrancia ! I really have only two suggestions, the rest are comments we may want to discuss or consider further.

**Review by DaniMori — APPROVED** [review]

**Review by Cristrinaranjus — APPROVED** [review]

**Review by robertomediavilla — APPROVED** [review]

**Review by robertomediavilla — APPROVED** [review]

**Review by JesusSierralaya — APPROVED** [review]

**Review by Cristrinaranjus — APPROVED** [review]

**Comment by robertomediavilla** [comment]
@DaniMori I think your approval is pending on this pull request. thanks!

File: output/Form.md [hunk 11]
```diff
@@ -57,22 +23,49 @@ title: Yerun Open Science Award form
...
-    wars, etc., research should (and will) move rapidly and collaboratively.
-    This requires training and collaboration, and it is the reason why we apply
-    for this grant.
-
-<!--# blank field, max 400 words -->
+As a research group and a World Health Organization's collaborating centre, we
+work daily with researchers and professionals living abroad. This requires tools
+that help tracking exchanges between labs in a comprehensive way, such as
+statistical code, manuscript drafts, or meeting minutes. By taking this
+approach, we aims not only to develop a stronger open science practice within
```
Comment (DaniMori) [inline output/Form.md:74]: "we _aim_"

File: output/Form.md [hunk 12]
```diff
@@ -57,22 +23,49 @@ title: Yerun Open Science Award form
...
+our lab, but also across different research consortia in the European Union
+(EU). We are currently in a test phase of implementing this open science way of
+working in our research group. This may potentially spread through different
+institutes in the region and improve reproducibility rates for the whole
```
Comment (DaniMori) [inline output/Form.md:78]: "institutions"?

File: output/Form.md [hunk 13]
```diff
@@ -57,22 +23,49 @@ title: Yerun Open Science Award form
...
+research community. For example,in the EU-funded project RESPOND, we have
```
Comment (DaniMori) [inline output/Form.md:79]: space after comma

File: output/Form.md [hunk 14]
```diff
@@ -57,22 +23,49 @@ title: Yerun Open Science Award form
...
+powerful tool that makes national and international collaboration possible is
+the Obiba Software. We currently use this open-source software on the RESPOND
+project to conduct federated analyses of different epidemiological cohort
+studies (software DataShield and R) while data is securely stored in local
+servers (software Opal). In this project, we also anonymised our datasets to
+comply with the General Data Protection Regulation of the European Union.
+Because a common anonymisation process across datasets was not possible (due to
+the sensitive nature of the data), we used scripts in R, Stata, and SPSS, based
+on each partners' preference, to keep the anonymisation as reproducible as
+possible at all study sites. One of the main benefits of developing this
```
Comment (DaniMori) [inline output/Form.md:99]: Empashize here that they are all "text-based", and thus susceptible to version control?

File: output/Form.md [hunk 15]
```diff
@@ -57,22 +23,49 @@ title: Yerun Open Science Award form
...
+practice is having a complete history accessible to all. This increases the
+transparency of a study and facilitates communication. Moreover, it is an
+interesting tool for the researcher himself. Indeed, having this history allowed
```
Comment (DaniMori) [inline output/Form.md:102]: "researchers themselves" (gender neutral)

File: output/Form.md [hunk 16]
```diff
@@ -57,22 +23,49 @@ title: Yerun Open Science Award form
...
+us to keep in writing what was done, what worked but also what led to errors.
+This written trace allowed us to save time and avoid reproducing the same
+mistake while understanding why. Also, when we want to remember the previous
```
Comment (DaniMori) [inline output/Form.md:105]: "mistakes"

**Review by DaniMori — APPROVED** [review]
My suggestions are only very minor and can be implemented in the final review of the template.

**Pull request merged**